src/Sylius/Bundle/CustomerBundle/README.md

```diff
@@ -0,0 +1,75 @@
+SyliusCustomerBundle [![Build status...](https://secure.travis-ci.org/Sylius/SyliusCustomerBundle.png?branch=master)](http://travis-ci.org/Sylius/SyliusCustomerBundle)
+==================
+
+[**Symfony2**](http://symfony.com) integration of Sylius Customer processing
+component.
+
+Sylius
+------
+
+**Sylius** - Modern ecommerce for Symfony2. Visit [Sylius.org](http://sylius.org).
+
+[phpspec](http://phpspec.net) examples
+--------------------------------------
+
+```bash
+$ composer install
+$ bin/phpspec run -f pretty
+```
+
+Documentation
+-------------
+
+Documentation is available on [**docs.sylius.org**](http://sylius.org/en/latest/bundles/SyliusCustomerBundle/index.html).
+
+Contributing
+------------
+
+All informations about contributing to Sylius can be found on [this page](http://docs.sylius.org/en/latest/contributing/index.html).
+
+Mailing lists
```

Comment:
No longer used, should be removed (and in separate PR from every other place also).